### PR TITLE
Human turn always first (#322)

### PR DESCRIPTION
**Pull request by vasilenkoalexey**
Fix https://github.com/ihhub/fheroes2/issues/322

### CODE REVIEW

File: src/fheroes2/game/game_startgame.cpp [hunk 1]
```diff
@@ -506,11 +506,18 @@ int Interface::Basic::StartGame( void )
     int res = Game::ENDTURN;
     const Players & players = conf.GetPlayers();
 
+    std::vector<Player *> rotate( players.size() );
```
Comment (ihhub) [inline src/fheroes2/game/game_startgame.cpp]: Let's call it `sortedPlayers` instead as rotate is a name of function in many namespaces.

**Comment by ihhub** [comment]
Hi @vasilenkoalexey I've changed the logic: all human turns go first as in the original game.

**Comment by vasilenkoalexey** [comment]
> I've changed the logic: all human turns go first as in the original game.

Hi! Yes, agreed.
But little notice (I'm not sure if this is important). There is INFO dialog in original game 

screenshot1

And if `Players` order chnaged it looks like
screenshot2

**Comment by ihhub** [comment]
Hmm, we could make a copy of Players, sort it while remaining the original order. What do you think?

**Comment by vasilenkoalexey** [comment]
It will be right

**Comment by vasilenkoalexey** [comment]
Ok, i see, now it clean.

**Comment by ihhub** [comment]
> Ok, i see, now it clean. But when copy `Players` class, function `Players::clear` is called twice, and trying delete twice `Player` objects. I think just `vector<Player *>` better.

Ah, okay, you're right. Could you please change the code. My apologies for not seeing such problem.

**Comment by ihhub** [comment]
Hi @vasilenkoalexey thank you very much for your valuable efforts for the project!

**Pull request merged**